TwilioVerifySDK/TwilioSecurity/Sources/Keychain/KeychainQuery.swift

```diff
@@ -52,7 +52,7 @@ struct KeychainQuery: KeychainQueryProtocol {
 
   /// Direct use of [kSecAttrAccessGroup](https://developer.apple.com/documentation/security/ksecattraccessgroup?language=swift),
   /// when set all methods will make use of it.
-  var accessGroup: String?
+  let accessGroup: String?
 
   init(accessGroup: String?) {
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/1230259fcee173b2bad2b801de0a522beecabdd9/TwilioVerifySDK/TwilioSecurity/Sources/Keychain/KeychainQuery.swift#L57"></span>This memberwise initializer would be synthesized automatically - you do not need to define it<br />
<code>unneeded_synthesized_initializer</code> <code>KeychainQuery.swift:57</code></td>
    </tr>
  </tbody>
</table>